lib/core/src/lib/form/components/widgets/core/form.model.spec.ts

```diff
@@ -276,19 +262,15 @@ describe('FormModel', () => {
     it('should raise validation event when validating form', () => {
         const form = new FormModel({}, null, false, formService);
 
-        formService.validateForm.subscribe((validateFormEvent) =>
-            expect(validateFormEvent).toBeTruthy()
-        );
+        formService.validateForm.subscribe((validateFormEvent) => expect(validateFormEvent).toBeTruthy());
```

Comment:
Another pre-existing issue, but these tests should be invoking the done() callback, since we are performing the assertions inside asynchronous subscribe blocks
```
it('should raise validation event when validating form', (done) => {
        const form = new FormModel({}, null, false, formService);

        formService.validateForm.subscribe((validateFormEvent) => { 
                expect(validateFormEvent).toBeTruthy());
                done();
        }
        form.validateForm();
});
```

Reply:
fixed